docs/tools_user_guide.md

```diff
@@ -277,15 +279,17 @@ Each device type has its own default runtime, as indicated below. Valid values i
 - `ort`: ONNX Runtime (default for `x86` device type).
 - `trt`: Nvidia TensorRT (default for `nvidia` device type).
 - `groq`: GroqFlow (default for `groq` device type).
-- [future] `pytorch1`: PyTorch 1.x-style eager execution.
-- [future] `pytorch2`: PyTorch 2.x-style compiled graph execution.
+- `torch`: PyTorch default execution.
+- `torch_compiled`: PyTorch 2.x-style compiled graph execution.
```

Comment:
Should this be `torch1.x` and `torch2.x`? Default execution is not really clear.

Reply:
Technically you can still do eager execution in torch2.x. IMO "torch_eager" and "torch_compiled" are the right names.

Reply:
Given the existence of graph mode, this might get a bit confusing. I'm consulting with our compiler team to define the clearest terminology here. 

So far, we have the following:
`torch - PyTorch default execution (either eager or graph mode, depending on how your model has been written).`

Reply:
`torch-eager` it is! This is the right terminology to use given that we are just invoking the `.forward` method of the `torch.nn.Module`.

Another interesting nit is that  `torch-compiled` could be a bit confusing, since `torch.jit` apis technically are compiling the graph. However, I tried to make it clear in the documentation that we are using `torch.compile()`.